content/en/docs/concepts/architecture/cri.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Container Runtime Interface (CRI)
+content_type: concept
+weight: 10
+---
+
+<!-- overview -->
+
+The Kubernetes Container Runtime Interface (CRI) defines the main
+[gRPC](https://grpc.io) protocol for the communication between the
+[cluster components](/docs/concepts/overview/components/#node-components)
+{{< glossary_tooltip text="kubelet" term_id="kubelet" >}} and
+{{< glossary_tooltip text="container runtime" term_id="container-runtime" >}}.
+
+The CRI is a plugin interface which enables the kubelet to use a wide variety of
+container runtimes, without having a need to recompile the cluster components.
+
+<!-- body -->
+
+## The API
+
+{{< feature-state for_k8s_version="v1.23" state="stable" >}}
+
+The latest version of the CRI is `v1`, which is being used by the kubelet as
+default expected version container runtimes have to fulfill. The kubelet acts as
+a client when connecting to the container runtime via gRPC. This means, if a
+container runtime does not support `v1`, then the kubelet tries to connect by
+using the previous version `v1alpha2`. If this connection fails as well, then
+the kubelet gives up. End users have therefore not to take action because the
+kubelet will automatically choose the correct CRI version to select.
```

Comment:
```suggestion
The kubelet acts as a client when connecting to the container runtime via gRPC.
For Kubernetes v{{< skew currentVersion >}}, the kubelet prefers to use CRI `v1`.
If a container runtime does not support `v1` of the CRI, then the kubelet tries to
negotiate any older supported version.
The v{{< skew currentVersion >}} kubelet can also negotiate CRI `v1alpha2`.
If the kubelet cannot negotiate a supported CRI version, the kubelet gives up
and doesn't register as a node.
```

I prefer this because it makes it more clear what support we're describing, and if
you end up looking at docs for a future Kubernetes release that doesn't support
`v1alpha2` it will be more obvious why not (we'd change this page for that release, of course).

If “doesn't register as a node” is wrong, let's document what the kubelet actually does in that situation
(eg fails to launch Pods, taints itself, whatever).